Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -56,6 +62,24 @@ public void Initialize(TelemetryConfiguration configuration)
                         {                            
                             this.telemetryConfiguration = configuration;
 
+                            if (!this.DoNotParseHttp && configuration != null)
```

Comment:
That was the main qq in this PR. Today when you do `new DependencyTelemetryModule()` you'll collect http dependenies as Azure Storage because Breeze will do transformation. Since logic moved from Breeze to SDK I was thinking it will be logical to register it be default.

Do you think it reasonable or we need to write release notes?